audius-cli

```diff
@@ -899,9 +900,7 @@ def deregister_plugin(ctx, name):
         return
     if name in plugins:
         plugins.remove(name)
-        # last plugin removed
-        if not plugins:
-            return
```

Comment:
if you removed the last plugin this actually wouldn't set the env var to empty, it would early return